docs/kubernetes/KubernetesPolicy.md

```diff
@@ -0,0 +1,62 @@
+# Programming Calico Policy in Kubernetes
+The [Calico-Kubernetes plugin](https://github.com/projectcalico/calico-docker/blob/master/docs/kubernetes/KubernetesIntegration.md) will allow you to program Networking policy directly from your Kubernetes pod config. 
```

Comment:
"Networking" should be lowercase.

This should probably read more like "Calico Kubernetes plugin allows you to specify networking policy on the Kubernetes API using pod annotations."

Side note: Don't stick the hyphen in Calico Kubernetes :)